### PR TITLE
Extract status message constants from AdminEmailComposePageAction and AdminEmailComposeSaveAction #4136

### LINKED ISSUE #4136: Extract status message constants from AdminEmailComposePageAction and AdminEmailComposeSaveAction
Extract "The requested email was not found" from AdminEmailComposePageAction and
"Email draft has been saved" from AdminEmailSendPageAction to Const.java

### PR DESCRIPTION
**Pull request by manishvishnoi2**
Fixes #4136

### CODE REVIEW

File: src/main/java/teammates/ui/controller/AdminEmailComposeSaveAction.java [hunk 1]
```diff
@@ -68,9 +68,9 @@ protected ActionResult execute() {
                                                         null);    
             data.emailToEdit.emailId = emailId;
         } else {
-            statusToAdmin = "Email draft has been saved: <br>" +
+            statusToAdmin = Const.StatusMessages.DRAFT_SAVED+": <br>" +
```
Comment (wkurniawan07) [inline src/main/java/teammates/ui/controller/AdminEmailComposeSaveAction.java]: Whitespace before/after the + sign

File: src/main/java/teammates/common/util/Const.java [hunk 2]
```diff
@@ -862,6 +862,9 @@
         public static final String IMAGE_TOO_LARGE = "The uploaded image was too large. ";
         public static final String FILE_NOT_A_PICTURE = "The file that you have uploaded is not a picture. ";
         public static final String NO_IMAGE_GIVEN = "Please specify a image to be uploaded.";
+        public static final String EMAIL_NOT_FOUND = "The requested email was not found";
+        public static final String EMAIL_NOT_FOUND_ADMIN = "The requested email for editing was not found";
```
Comment (wkurniawan07) [inline src/main/java/teammates/common/util/Const.java]: Use just one of them, they're essentially the same thing.

**Comment by wkurniawan07** [comment]
@manishvishnoi2 I added some comments to work on.

**Comment by manishvishnoi2** [comment]
@wkurniawan07 Sir i submitted the patch with changes.

File: src/main/java/teammates/common/util/Const.java [hunk 3]
```diff
@@ -862,6 +862,8 @@
         public static final String IMAGE_TOO_LARGE = "The uploaded image was too large. ";
         public static final String FILE_NOT_A_PICTURE = "The file that you have uploaded is not a picture. ";
         public static final String NO_IMAGE_GIVEN = "Please specify a image to be uploaded.";
+        public static final String EMAIL_NOT_FOUND = "The requested email was not found";
+        public static final String DRAFT_SAVED = "Email draft has been saved";
```
Comment (wkurniawan07) [inline src/main/java/teammates/common/util/Const.java]: `EMAIL_DRAFT_SAVED`

**Comment by wkurniawan07** [comment]
One more minor change and it should be good to go.

**Comment by manishvishnoi2** [comment]
@wkurniawan07  Sir i changed the constant name from DRAFT_SAVED to EMAIL_DRAFT_SAVED

**Comment by wkurniawan07** [comment]
Looks good. Thanks for your contribution @manishvishnoi2 :+1:

**Comment by manishvishnoi2** [comment]
@wkurniawan07  Thank you sir for reviewing and helping me.

**Pull request merged**